ledger/participant-integration-api/src/main/scala/platform/store/DbType.scala

```diff
@@ -13,7 +13,7 @@ private[platform] sealed abstract class DbType(
     if (supportsParallelWrites) maxConnections else 1
 }
 
-private[platform] object DbType {
+object DbType {
```

Comment:
What is the benefit of this?

Reply:
Unrelated to this PR: As you pointed out, Marcin, this will also help simplify #11577

Reply:
goes together with the DbDispatcher pull-up: needed for creating those. (actually as soon as sandbox-classic is removed we can seal that back to platform if we want)